pkg/k8s/watchers/namespace.go

```diff
@@ -139,7 +149,8 @@ func (u *namespaceUpdater) update(newNS *slim_corev1.Namespace) error {
 	for _, ep := range eps {
 		epNS := ep.GetK8sNamespace()
 		if newNS.Name == epNS {
-			err := ep.ModifyIdentityLabels(labels.LabelSourceK8s, newIdtyLabels, oldIdtyLabels)
+			err := ep.ModifyIdentityLabelsWithJitter(labels.LabelSourceK8s, newIdtyLabels, oldIdtyLabels,
+				randomDuration(option.Config.CiliumIdentityMaxJitter))
```

Comment:
consider: set as a field in the namespaceUpdater rather than referencing the global config